test/unit/lodash.spec.js

```diff
@@ -0,0 +1,522 @@
+'use strict';
+
+var chai = require('chai');
+var expect = chai.expect;
+var _ = require('lodash');
+var datasets = require('../../app/datasets');
+var Guy = require('./Guy');
+var chance = require('chance').Chance();
+
+describe('Lodash training', function ()
+{
+    describe('Math functions', function () {
+        describe('add', function () {
+
+            var params, capacity, area;
+
+            beforeEach(function () {
+                params = datasets.add();
+                capacity = params[0];
+                area = params[1];
+            });
+
+            it('should verify params', function () {
+                expect(capacity instanceof Function && area instanceof Function).to.eql(true);
```

Comment:
Those exercises are not meant to test one's knowledge about javascript in general(as they expect applicant to be knowlegable just enough) but lodash understanding and simple use cases. Add function in general is so basic that it only accepts numbers. The only one case this would be different I think would be having floats instead. Functions are not acceptable.